lib/avo/base_resource.rb

```diff
@@ -48,6 +48,7 @@ class BaseResource
     class_attribute :record_selector, default: true
     class_attribute :keep_filters_panel_open, default: false
     class_attribute :extra_params
+    class_attribute :link_to_child_resource, default: false
```

Comment:
Do we still need this attribute on the base resource? I see we're not mutating it anywhere.

Reply:
Related to my other comment. Its not mutated. But its hold the resource global configuration.
if enabled ( value: true)